ui/apps/everest/src/components/cluster-form/advanced-configuration/advanced-configuration-schema.ts

```diff
@@ -55,6 +55,24 @@ export const advancedConfigurationsSchema = () =>
           });
         }
       });
+      const seen = new Set<string>();
+      sourceRanges.forEach(({ sourceRange }, index) => {
+        if (sourceRange && seen.has(sourceRange)) {
```

Comment:
I would use this approach, instead of having a set for seen 
```suggestion
         const uniqueRanges = new Set(
        Object.values(sourceRanges).map((item) => item.sourceRange)
      );
      if (uniqueRanges.size !== sourceRanges.length) { ...
```